include/cucumber-cpp/internal/CukeEngine.hpp

```diff
@@ -59,7 +61,7 @@ class PendingStepException : public InvokeException {
  * It uses standard types (as much as possible) to be easier to call.
  * Returns standard types if possible.
  */
-class CukeEngine {
+class CUCUMBER_CPP_EXPORT CukeEngine {
```

Comment:
I believe all of the members of this class are used internally only, with the exception of the constructor and destructor. So if you explicitly declare those and add (empty) implementations to the .cpp file then exporting just those should be enough.

I know on GNU/Linux that would work. Could you verify that that works on MSVC too? I.e. only exporting some members of a class without exporting the entire class?